std/process.d

```diff
@@ -3251,16 +3293,20 @@ private:
     // New variable
     environment["std_process"] = "foo";
     assert (environment["std_process"] == "foo");
+    assert ("std_process" in environment);
```

Comment:
See https://github.com/dlang/phobos/pull/5169 for an enforcement and automatic unification of this `assert` style ;-)